device-triage/triage.sh

```diff
@@ -21,8 +21,8 @@ if [ -z "${RPI_DEVICE_BOOTLOADER_CONFIG_FILE}" ]; then
     RPI_DEVICE_BOOTLOADER_CONFIG_FILE=/var/lib/rpi-sb-provisioner/bootloader.default
 fi
 
-if [ -z "${CUSTOMER_KEY_FILE_PEM}" ]; then
-    echo "You must provide a customer key file in the environment via CUSTOMER_KEY_FILE_PEM"
+if [ -z "${CUSTOMER_KEY_FILE_PEM}" ] || [ -z "${CUSTOMER_KEY_PKCS11_NAME}" ]; then
```

Comment:
I believe this should be `&&`, i.e. if neither of them are set we've got an error condition.